lib/utils/classes.py

```diff
@@ -21,3 +26,157 @@ class Stock:
 
     def __init__(ticker: str):
         self.ticker = ticker
+
+
+
+class myLoginSession:
+    """
+    Usage: 
+    LS = myLoginSession('user', 'pass')
+    session = LS.mySession()
+
+    """
+    __username: str
+    __password: str
+
+    __login_url: str = 'https://www.screener.in/login/'
+
+    __session: [requests.sessions.Session]
+
+    def __init__(self, username, password):
+        #TODO: Remove un necessary stuff. See if we need to store theusername and password in class variables
+        self.__username = username
+        self.__password = password
+        self.__login()
+
+    def __login(self):
+        self.__session = requests.Session()
+        self.__session.get(self.__login_url)
+        csrftoken = self.__session.cookies['csrftoken']
+
+        #Appending the csrftoken to the data payload
+        login_payload = {'username':self.__username, 'password':self.__password, 'csrfmiddlewaretoken':csrftoken}
+        resp = self.__session.post(self.__login_url, data = login_payload, headers=dict(Referer=self.__login_url))
+        
+    def mySession(self) -> [requests.sessions.Session]:
+        return self.__session
+
+    def closeSession(self):
+        self.__session.close()
+
+
+
+class Companies:
```

Comment:
Why does this need to be a class?

Also, All of this code should be a part of `lib/fetcher.py` and not this file.

Reply:
Initially, I wanted to but if we want to implement more filtering at the fetch stage then having a separate class is better.

Reply:
Isn’t fetch only supposed to fetch a list of stocks? We don’t need any metadata (or at most, minimal metadata) at this stage